tests/unit/server/api/test_subscriptions.py

```diff
@@ -227,7 +227,7 @@ async def test_openai_text_response_emits_expected_payloads_and_records_expected
         assert not input
         assert "api_key" not in input_value
         assert "apiKey" not in input_value
-        assert (output := span.pop("output")).pop("mimeType") == "json"
+        assert (output := span.pop("output")).pop("mimeType") == "text"
```

Comment:
these were different before, I ctually don't know why they are different but they are not the openinference values, this span is coming from subscription_span which comes from:
```python
  subscription_span = last_payload["chatCompletion"]["span"]
        span_id = subscription_span["id"]

        # query for the span via the node interface to ensure that the span
        # recorded in the db contains identical information as the span emitted
        # by the subscription
        data = await gql_client.execute(
            query=self.QUERY, variables={"spanId": span_id}, operation_name="SpanQuery"
        )
        span = data["span"]
        assert json.loads(attributes := span.pop("attributes")) == json.loads(
            subscription_span.pop("attributes")
        )
        attributes = dict(flatten(json.loads(attributes)))
        assert span == subscription_span
        ```

the pop of the attributes below comes from the span attributes. Maybe you know whats going on here, but going to keep as text for now since it works